baseline/OSPS-QA.yaml

```diff
@@ -64,6 +64,12 @@ controls:
           - 6.2.1
           - 6.5.1
           - 6.5.2
+      - reference-id: 800-161
+        identifiers:
+          - PL-8
```

Comment:
I'm not sure about this one. This Baseline control isn't really about "security and privacy architectures"